BlueMapCore/src/main/java/de/bluecolored/bluemap/core/world/mca/chunk/Chunk_1_13.java

```diff
@@ -273,6 +289,7 @@ public static class Level {
         private HeightmapsData heightmaps = new HeightmapsData();
         private SectionData @Nullable [] sections = null;
         private int[] biomes = EMPTY_INT_ARRAY;
+        @NBTName("TileEntities") private List<Map<String, Object>> blockEntities = List.of(Map.of());
```

Comment:
With the `TypeDeserializer<BlockEntity>` mentioned above, this can then be simplified into a `List<BlockEntity>` :)

Reply:
Done